### DISCUSSION TITLE
How do you use Understrap? Starter Theme vs Parent Theme Framework

### DISCUSSION THREAD

**Discussion by bacoords** [holger1411/understrap#1291] [General]
Open question for all contributors/users/casual browsers:

How do you use Understrap? Do you 

1. Use Understrap as a starter theme, make your changes to it, and turn off dotOrg theme updates to keep your customizations?
2. Grab Understrap as a parent theme and use [Understrap Child](https://github.com/understrap/understrap-child) to make your theme, allowing updates to the parent theme from dotOrg?
3. Run Understrap as a parent theme. The end.

It feels like the entire WP ecosystem is in favor of having a parent theme, with all custom development in the child theme. And yet _Underscores was explicitly not a parent theme. There's a number of developer-centric features that I'd love to implement, but feel like child-theme territory. Just curious for more perspective.

**Comment by danemorgan** [comment]
The way I used Underscores was sort of no. 1, but my method of disabling updates was to rename the theme. Which they eventually made easy with the tool at underscores.me.

For the other themes I've used ( almost exclusively Theme hybrid and Genesis ), I always installed as a parent and then made a child for it.

**Comment by Thomas-A-Reinert** [comment]


**Reply by danemorgan** [reply]
Maybe a better question is which path would you be following if the world was a perfect place?

**Comment by LukaszJaro** [comment]
I don't like child themes, feel they add more complexity and bloat. I used understrap as (1)starter theme until I noticed the repo was looking abandoned with no future/gutenberg support so switched to wprig.

I perform manual updates like updating bootstrap. I mean if you're doing some heavy customization or following some mockups with specific functionality, it becomes a unique theme and no need for child theme which can also break things.

Congrats to the new owner and maintainers for bringing understrap back! Will have another look at understrap for my next project.

**Comment by gettonet** [comment]
Use Understrap as a starter theme, run search/replace to call it the same as website it's on, and continue customization. 
Since adding the PostCSS Understrap Palette Generator dependency in package.json, all that do the same need to be extra careful since running search/replace on the whole thing will also replace the path to this dependency, which contains "understrap" string:
`"postcss-understrap-palette-generator": "git+https://github.com/bacoords/postcss-understrap-palette-generator.git",`
